packages/eas-cli/src/build/context.ts

```diff
@@ -30,6 +31,7 @@ export interface BuildContext<T extends Platform> {
   accountName: string;
   buildProfile: BuildProfile<T>;
   buildProfileName: string;
+  resourceClass?: BuildResourceClass;
```

Comment:
This shouldn't be optional here. We already know the exact value after converting user input to graphql enum.